software/java/en.md

```diff
@@ -14,42 +14,42 @@ You can set up Java by following the instructions below:
 
 ## JRE
 
-Grab the Java Runtime Environment (JRE) as `.tar.gz` from the [Oracle Download Page](https://www.oracle.com/technetwork/java/javase/downloads/jre8-downloads-2133155.html).
+Grab the newest Java Runtime Environment (JRE) as `.tar.gz` from the [Oracle Download Page](https://www.oracle.com/technetwork/java/javase/downloads/jre8-downloads-2133155.html).
 
 Extract JRE and move it to /opt:
 
 ``` bash
 cd ~/Downloads
-tar xf jre-8u181-linux-x64.tar.gz
+tar xf jre-8u202-linux-x64.tar.gz
 sudo mkdir -p /opt
-sudo mv jre1.8.0_181 /opt/
-sudo ln -svf /opt/jre1.8.0_181/bin/java /usr/bin/java
+sudo mv jre1.8.0_202 /opt/
+sudo ln -svf /opt/jre1.8.0_202/bin/java /usr/bin/java
 ```
 
 To enable Java in Firefox:
 
 ``` bash
 mkdir -p ~/.mozilla/plugins
-ln -s /opt/jre1.8.0_181/lib/amd64/libnpjp2.so ~/.mozilla/plugins/libnpjp2.so
+ln -s /opt/jre1.8.0_202/lib/amd64/libnpjp2.so ~/.mozilla/plugins/libnpjp2.so
 ```
 
 ## JDK
 
-Grab the Java Development Kit (JDK) as `.tar.gz` from the [Oracle Download Page](http://www.oracle.com/technetwork/java/javase/downloads/jdk8-downloads-2133151.html).
+Grab the newest Java Development Kit (JDK) as `.tar.gz` from the [Oracle Download Page](http://www.oracle.com/technetwork/java/javase/downloads/jdk8-downloads-2133151.html).
 
 Extract JDK and move it to /opt:
 
 ``` bash
 cd ~/Downloads
-tar xf jdk-8u181-linux-x64.tar.gz
+tar xf jdk-8u202-linux-x64.tar.gz
```

Comment:
Correct below changes with similar modification that I requested be made for JRE.

Reply:
Done in the same way as above, thanks.